src/redux/reducers.js

```diff
@@ -1020,11 +1020,14 @@ export function users(state = {}, action) {
       return mergeByIds(state, (action.payload.users || []).map(userParser))
     }
     case ActionTypes.REALTIME_POST_NEW:
+    case ActionTypes.REALTIME_LIKE_NEW:
     case ActionTypes.REALTIME_COMMENT_NEW: {
-      return mergeByIds(state, (action.users || []).map(userParser))
-    }
-    case ActionTypes.REALTIME_LIKE_NEW: {
-      return mergeByIds(state, ([action.user]).map(userParser))
+      const safeUsers = action.users || [{}]
+      const userAlreadyAdded = state[safeUsers[0].id]
+      if (userAlreadyAdded) {
+        return state
+      }
+      return mergeByIds(state, safeUsers.map(userParser))
```

Comment:
So the `mergeByIds(state, safeUsers.map(userParser))` will just return `state` because of id absence? OK, it's quite cumbersome to understand promptly, but I'm fine with having it if it's fine for you.